File: Programmers/pro_12940_최대공약수와 최소공배수.js
function solution(n, m) {
    const gcd = ((a, b) => {
        let r = 0
        r = parseInt(a % b)
        while (b) {
            r = parseInt(a % b)
            a = b
            b = r
        }
        return a
    })(n, m)
    return [gcd, n * m / gcd];
}

File: Programmers/pro_12940_최대공약수와 최소공배수.cpp
#include <string>
#include <vector>

using namespace std;

int getGcd(int a, int b){
    int r=1;
    while(b){
        r = a % b;
        a = b;
        b = r;       
    }
    return a;
}

vector<int> solution(int n, int m) {    
    vector<int> answer;
    int gcd = getGcd(n,m);
    answer.push_back(gcd);
    answer.push_back(n*m/gcd);
    return answer;
}